### PR TITLE
Add `proptest`'s regressions to `.gitignore`

### PR DESCRIPTION
**Pull request by upbqdn**
This is a quick update to the `.gitignore` file so that `git` ignores regressions generated by `proptest`.

## Review
Anyone can review.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+# Vscode detritus
 .vscode/
 .zebra-state/
 .cargo/
+
 # Nix configs
 shell.nix
+
+# Proptest cache
+*/proptest-regressions/
```
Comment (oxarbitrage) [inline .gitignore]:
```suggestion
*/proptest-regressions/

```

The previous version had a newline at the end.

**Review by oxarbitrage — DISMISSED** [review]
I committed the regressions to my branch a few times in the past. I think this is a good change that we should do.

**Review by oxarbitrage — APPROVED** [review]

**Review by conradoplg — CHANGES_REQUESTED** [review]
I'm not sure about this, there are legitimate reasons for adding to the proptest regressions (I think we should always commit them, unless it was a bug in the test itself), and this makes harder to detect when a regression was generated (it won't show up in `git status`) and harder to commit (will need `--force`)

**Comment by upbqdn** [comment]
Ok, I wrongly assumed the regressions shouldn't be committed. I should have read the docs before I submitted this. The docs mention that those files should be added to source control. Thank you for the note @conradoplg. Closing.

**Pull request closed without merging**